src/hooks/useValidation.js

```diff
@@ -0,0 +1,6 @@
+import { useCallback } from 'react';
+import validate from 'validate.js';
+
+validate.validators.presence.options = { allowEmpty: false };
+
+export default constraints => useCallback(values => validate(values, constraints), [constraints]);
```

Comment:
You are not allowing for the user to pass options to `validate`  function. At the top of my head, I can think of a common option we pass: `fullMessages: false` to disable that the field name is prepended to the error message